src/typescript/azure-openapi-validator/tests/resources/utilities/exception.json

```diff
@@ -0,0 +1,1177 @@
+{
```

Comment:
prefer to use small example and change file name `exception.json` more specific.